src/odemis/driver/xt_client.py

```diff
@@ -0,0 +1,368 @@
+        """
+        Setting the spot size of the ebeam.
+        Parameters
+        ----------
+        spotsize: float
+            desired spotsize, unitless
+        """
+        self.client.set_ebeam_spotsize(spotsize)
+
+    def get_ebeam_spotsize(self):
+        """Returns: (float) the current spotsize of the electron beam (unitless)."""
+        return self.client.get_ebeam_spotsize()
+
+    def spotsize_info(self):
+        """Returns: (dict) the unit and range of the spotsize. Unit is None means the spotsize is unitless."""
+        return self.client.spotsize_info()
+
+    def set_dwell_time(self, dwell_time):
+        """
+
+        Parameters
+        ----------
+        dwell_time: float
+            dwell time in seconds
+        """
+        self.client.set_dwell_time(dwell_time)
+
+    def get_dwell_time(self):
+        """Returns: (float) the dwell time in seconds."""
+        return self.client.get_dwell_time()
+
+    def dwell_time_info(self):
+        """Returns: (dict) range of the dwell time and corresponding unit."""
+        return self.client.dwell_time_info()
+
+    def set_ht_voltage(self, voltage):
+        """
+        Set the high voltage.
+
+        Parameters
+        ----------
+        voltage: float
+            Desired high voltage value in volt.
+
+        """
+        self.client.set_ht_voltage(voltage)
+
+    def get_ht_voltage(self):
+        """Returns: (float) the HT Voltage in volt."""
+        return self.client.get_ht_voltage()
+
+    def ht_voltage_info(self):
+        """Returns: (dict) the unit and range of the HT Voltage."""
+        return self.client.ht_voltage_info()
+
+    def blank_beam(self):
+        """Blank the electron beam."""
+        self.client.blank_beam()
+
+    def unblank_beam(self):
+        """Unblank the electron beam."""
+        self.client.unblank_beam()
+
+    def beam_is_blanked(self):
+        """Returns: (bool) True if the beam is blanked and False if the beam is not blanked."""
+        return self.client.beam_is_blanked()
+
+    def pump(self):
+        """Pump the microscope's chamber. Note that pumping takes some time. This is blocking."""
+        self.client.pump()
+
+    def get_vacuum_state(self):
+        """Returns: (string) the vacuum state of the microscope chamber to see if it is pumped or vented."""
+        return self.client.get_vacuum_state()
+
+    def vent(self):
+        """Vent the microscope's chamber. Note that venting takes time (appr. 3 minutes). This is blocking."""
+        self.client.vent()
+
+    def get_pressure(self):
+        """Returns: (float) the chamber pressure in pascal."""
+        return self.client.get_pressure()
+
+    def home_stage(self):
+        """Home stage asynchronously. This is non-blocking."""
+        self.client.home_stage()
+
+    def is_homed(self):
+        """Returns: (bool) True if the stage is homed and False otherwise."""
+        return self.client.is_homed()
+
+    def get_channel(self, name='electron1'):
```

Comment:
For all these functions related to channel, I'm not convinced it's useful to have a default value for the name. Without passing the name explicitly, the client code is going to seem weird. It's better to force the client code to always pick the channel. Please add a documentation for the list of possible names (and maybe even add some constants at the top of this module, corresponding to each allowed value).